### DISCUSSION TITLE
How do i make my audio go into 1 channel?

### DISCUSSION THREAD

**Discussion by Dal1ster** [alvr-org/ALVR#572] [General]
when i try to run the latest version of alvr, it errors out saying cannot have 2 channels of audio at once. How do i make it so it goes in 1 channel? My Computer headset is G635 Gaming headset, and the vr headset is the quest 2.

**Comment by JosNun** [comment]
ALVR currently only supports stereo sound (dual channel, left and right). The G635 has surround sound, so ALVR can't really "talk" to it. Do you have another audio device you could output to? If not, you may be able to use VB-Cable, and have steamVR output to / ALVR read from it.
